state-chain/pallets/cf-ingress-egress/src/migrations/add_refund_params.rs

```diff
@@ -76,10 +77,12 @@ impl<T: Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
 					old::ChannelAction::CcmTransfer {
 						destination_asset,
 						destination_address,
+						broker_fees,
 						channel_metadata,
 					} => ChannelAction::CcmTransfer {
 						destination_asset,
 						destination_address,
+						broker_fees,
```

Comment:
Isn't the more correct thing to do here to delete the migration? It'll be applied on 1.5 so we don't need to update this

Reply:
This is a hotfix and should go directly into 1.5 I believe. My understanding (from chatting with @dandanlen) is that we will integrate this fix (in some form) directly into https://github.com/chainflip-io/chainflip-backend/pull/5106.